index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>first capstone project</title>
+</head>
+<body>
+    <header> 
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse design templates as closely as possible. Let's try our best to stick to the given design. Kindly add the dark toolbar on the top of the navigation to match the template. 😎️
- [ ] Please give a bit more spacing on the sides of the navigation bar. 🤝️

**Your design**
![image](https://user-images.githubusercontent.com/17243325/217836499-4004540d-30fa-46a2-ba6b-6aab15799b3d.png)

**Template**
![Screenshot from 2023-02-09 16-42-19](https://user-images.githubusercontent.com/17243325/217835897-f10ff360-a06f-40fd-9667-ee9036c79992.png)